20150310-wp-core-contrib-day.md

```diff
@@ -0,0 +1,255 @@
+
+![Modify Ticket Block](https://s3.amazonaws.com/cdn.deliciousbrains.com/content/uploads/2015/03/Modify-Ticket-Block.png)
+
+Depending on what you have done or found, you may need to alter some of the Workflow Keywords within the **Change Properties** section.
+
+In most cases you'll be removing things like the "needs-patch" or "needs-refresh" keywords and adding the "has-patch" and "needs-testing" keyword. Once done this will move the ticket into someones keyword based report as something to review and test. Don't forget to check out the ["how we use our bug tracker"](https://make.wordpress.org/core/handbook/trac/) article if in doubt as to how to change the keywords.
+
+You're very unlikely to need to change anything within the **Action** section, I mistakenly changed some items there once, I won't be doing that again in a hurry! This section is basically for closing a case, either because it has been completed and merged into the repository or because it has been found to be invalid for one reason or another including being a duplicate of another. It's probably best to leave this section alone unless you're on the WordPress Core team or have been assigned to review the ticket's validity.
+
+### Get Clean Again
+Now that you've submitted your patch and are ready to move on to your next ticket, ensure that your repository checkout is ready for fresh development (see [Keep It Clean][]).
+
+There is nothing wrong with leaving the patch file you created in the root directory for a while though, it should not interfere with any new development as it should be automatically excluded from any diffs, and it's handy to keep it around in case you need to re-work the patch later (although you can of course re-download your submitted patch file from the ticket).
+
+## Keeping On Top Of Your Tickets
+Even though you've submitted the best patch in the history of WordPress Core development, it doesn't mean you can sit back and relax, for any number of reasons your patch might not get reviewed, tested and committed through to trunk. You should keep tabs on your tickets and every now and then ensure they still apply and refresh the patches if they haven't gone anywhere yet. You may also get notifications by email when something changes on the tickets you have worked on.
+
+### Email Notifications
+There's a good chance that you'll get a few notifications by email when things change on the tickets you've worked on, make sure you read these as there may be things you need to do in response, such as re-work your patch or reply to a comment to clarify something.
+
+The best kind of notification is a commit notification, or what I call a "props" email. These are awesome, it's great to see your username given some props when a ticket is closed as complete, take a moment and pat yourself on the back, you earned it!
+
+### [My Patches](https://core.trac.wordpress.org/my-patches)
```

Comment:
Worth checking if our blog formatting makes it clear if an h3 as a link is clear enough. If not might want to show the link below.